src/features/projects/components/maps/Credits.tsx

```diff
@@ -44,6 +44,20 @@ export default function Credits({ setCurrencyCode }: Props): ReactElement {
                     {`🌐 ${language ? language.toUpperCase() : ''
                         } • ${selectedCurrency}`}
                 </div>
+                <a
+                    rel="noopener noreferrer"
+                    href={`https://status.plant-for-the-planet.org/`}
+                    target={'_blank'}
+                >
+                    {t('common:status')}
```

Comment:
The status item is also not part of our bigger footer? Should it or shouldn't it?

Big footer:

<img width="836" alt="Bildschirmfoto 2021-04-05 um 12 16 29" src="https://user-images.githubusercontent.com/1532418/113563991-e45ae900-9608-11eb-88b0-6c047e7d7d33.png">

Small footer:

<img width="512" alt="Bildschirmfoto 2021-04-05 um 12 16 40" src="https://user-images.githubusercontent.com/1532418/113563998-e58c1600-9608-11eb-9555-d893cbf51f96.png">

Reply:
@harshvitra

Reply:
Yes we might add it to the bigger footer also